engine/src/CalciteInterpreter.cpp

```diff
@@ -686,40 +730,51 @@ blazing_frame evaluate_split_query(std::vector<ral::io::data_loader> input_loade
 			}
 		}
 		// these shoudl be split up and run on different threads
-		blazing_frame left_frame;
-		left_frame = evaluate_split_query(input_loaders,
+		std::unique_ptr<ral::frame::BlazingTable> left_frame = evaluate_split_query(input_loaders,
```

Comment:
put these on different threads. They used to be on different threads I have no clue when that went away.

Reply:
that never has run on different threads. it would be a good idea, but lets leave that for bigger than gpu

Reply:
good idea we will do that for bigger than gpu